src/core/public/chrome/chrome_service.test.ts

```diff
@@ -0,0 +1,239 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import * as Rx from 'rxjs';
+import { toArray } from 'rxjs/operators';
+
+const store = new Map();
+(window as any).localStorage = {
+  setItem: (key: string, value: string) => store.set(String(key), String(value)),
+  getItem: (key: string) => store.get(String(key)),
+  removeItem: (key: string) => store.delete(String(key)),
+};
+
+import { ChromeService } from './chrome_service';
+
+beforeEach(() => {
+  store.clear();
+});
+
+describe('start', () => {
+  describe('brand', () => {
+    it('updates/emits the brand as it changes', async () => {
+      const service = new ChromeService();
+      const start = service.start();
+      const promise = start
+        .getBrand$()
+        .pipe(toArray())
+        .toPromise();
+
+      start.setBrand({
+        logo: 'big logo',
+        smallLogo: 'not so big logo',
+      });
+      start.setBrand({
+        logo: 'big logo without small logo',
+      });
+      service.stop();
+
+      await expect(promise).resolves.toMatchInlineSnapshot(`
```

Comment:
Is there an advantage to using `toMatchInlineSnapshot` over `toEqual`? Personally I would find the latter more readable:

```js
      await expect(promise).resolves.toEqual([
        {},
        {
          logo: "big logo",
          smallLogo: "not so big logo",
        },
        {
          logo: "big logo without small logo",
          smallLogo: undefined,
        },
      ]);
    });
```

Reply:
Mostly maintenance, being able to update the snapshots with `-u` and automatically update them with `--watch` are pretty huge benefits to me